packages/utils/strings.ts

```diff
@@ -1,10 +1,6 @@
 import { capitalize as toCapitalize } from '@vue/shared'
 
-export {
-  camelize,
-  hyphenate,
-  hyphenate as kebabCase, // alias
-} from '@vue/shared'
+export { camelize, hyphenate } from '@vue/shared'
```

Comment:
Maybe we could add `const kebabCase = hyphenate`

If someone has introduced `hyphenate` in element-plus, deleting it directly may result in an error.